docs/content/preview/explore/ycql-language/keyspaces-tables.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Keyspaces and tables
+linkTitle: Keyspaces and tables
+description: Keyspaces and tables in YCQL
+image: /images/section_icons/secure/create-roles.png
+menu:
+  preview:
+    identifier: explore-ycql-language-features-keyspaces-tables
+    parent: explore-ycql-language
+    weight: 60
+type: docs
+---
+
+This section explores the topics of Keyspaces and Tables in YCQL using the command line shell
+`ycqlsh`.
+
+{{% explore-setup-single %}}
+
+## YCQL shell
+
+Use the [ycqlsh shell](../../../admin/ycqlsh/) to interact with a Yugabyte database cluster using the [YCQL API](../../../api/ycql/). Because `ycqlsh` is derived from the Apache Cassandra shell `cqlsh` code base, most `cqlsh` commands work as is in `ycqlsh`. Unsupported commands are listed towards the end of this page.
```

Comment:
```suggestion
Use the [ycqlsh shell](../../../admin/ycqlsh/) to interact with a Yugabyte database cluster using the [YCQL API](../../../api/ycql/). Because `ycqlsh` is derived from the Apache Cassandra shell [cqlsh](https://cassandra.apache.org/doc/latest/cassandra/tools/cqlsh.html), most `cqlsh` commands work as is in `ycqlsh`. Unsupported commands are [listed](#unsupported-cqlsh-commands) at the end of this page.
```